pallets/order-book/src/benchmarking.rs

```diff
@@ -23,8 +23,8 @@ use sp_runtime::{traits::checked_pow, FixedPointNumber};
 
 use super::*;
 
-const CURRENCY_IN: u32 = 1;
-const CURRENCY_OUT: u32 = 2;
+const CURRENCY_IN: u32 = 1000;
```

Comment:
Out of curiosity, what was the issue?

Reply:
Benchmarks failed because at least currency in existed in storage already.

Reply:
Mmm... but who was registering that `CURRENCY_IN` before? 🤔

Reply:
Couldn't find a quick answer and didn't wanna spend more time debugging this because debugging runtime benchmarks is really annoying.

Reply:
No problem! Just curious in case you knew it